### ISSUE TITLE
Binding keyboard shortcuts to common actions 

### ISSUE THREAD

**Issue by abrac** [jnwillits/The-Flashcard-Project#3]
It would nice to have keyboard shortcuts that the user can define for common actions.

**Comment by jnwillits** [comment]
Agreed. I am busy with other things now, but I will welcome your contributions!

**Comment by abrac** [comment]
Okay, I'll work on it. I've still got to read through your code, but will submit a pull request once I'm done with the task. You can assign me to this issue.
